nuget/Microsoft.NETCore.Runtime.Mono.LLVM.Sdk/runtime.Windows_NT.Microsoft.NETCore.Runtime.Mono.LLVM.Sdk.props

```diff
@@ -1,6 +1,8 @@
 <?xml version="1.0" encoding="utf-8"?>
 <Project>
   <ItemGroup>
+    <File Include="$(ProjectDir)\clang\bindings\python\clang\**" TargetPath="bindings\python\clang\" />
+    <File Include="$(_LLVMInstallDir)\bin\libclang.dll" TargetPath="tools\$(PackageTargetRuntime)\bin\" Condition="'$(TargetArchitecture)' != 'arm' and '$(TargetArchitecture)' != 'arm64'" />
```

Comment:
Why not arm?

Reply:
There's an error when building on arm and as we don't need it for wasm I just disabled it. Maybe I can try to fix it on Raspberry at some point.

Reply:
The error on arm:
```
  [ 24%] Building arm_neon.h...
  ../../../../bin/clang-tblgen: 1: ../../../../bin/clang-tblgen: ELF�p�@@�@8: not found
  ../../../../bin/clang-tblgen: 3: ../../../../bin/clang-tblgen: I*&AW: not found
  ../../../../bin/clang-tblgen: 1: ../../../../bin/clang-tblgen: Syntax error: Unterminated quoted string
  tools/clang/lib/Headers/CMakeFiles/clang-resource-headers.dir/build.make:862: recipe for target 'tools/clang/lib/Headers/arm_neon.h' failed
  CMakeFiles/Makefile2:15360: recipe for target 'tools/clang/lib/Headers/CMakeFiles/clang-resource-headers.dir/all' failed
  make[2]: *** [tools/clang/lib/Headers/arm_neon.h] Error 2
  make[1]: *** [tools/clang/lib/Headers/CMakeFiles/clang-resource-headers.dir/all] Error 2
  make[1]: *** Waiting for unfinished jobs....
```